.github/workflows/docs-build.yml

```diff
@@ -20,10 +20,12 @@ jobs:
         with:
           node-version: lts/*
       - name: Build documentation
+        env:
+          PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
```

Comment:
Does loading it into an environment variable here offer any value, given that this can be modified by external contributors anyway?

Reply:
I think you're probably right here, but it could be good to start the practice of always using `env` to automatically interpolate input values.